### ISSUE TITLE
waiting for locator("id=loginUsername")

### ISSUE THREAD

**Issue by Thorben-the-IT** [aahashemi/YouTubeAutomation-Reddit#29]
![Bildschirmfoto vom 2024-03-01 15-05-27](https://github.com/aahashemi/YouTubeAutomation-Reddit/assets/161852406/f4dca928-ed84-44b5-b302-5a5f5044fbfd)

just these faliures and i dont know how to fix it.

**Comment by Zenefff** [comment]
Same issue

**Comment by alexvsc5** [comment]
Got the exact same! Anyone got solution yet?

**Comment by thepede** [comment]
Same exact error - I haven't found anything yet. I'm not an experienced programmer so it's unlikely I'll find anything.

**Comment by nssharmaofficial** [comment]
@Thorben-the-IT @alexvsc5 @thepede @Zenefff I fixed the issue. Check working solution [here](https://github.com/nssharmaofficial/reddit-hole) since this does not seem to be active.

**Comment by thepede** [comment]
You are truly too kind. Thank you!

**Comment by AIlhomov** [comment]
have the same issue, i tried debugging and i get an png file back and it gets stuck at a page that says "whoa there, pardner! Your request has been blocked due to an network policy."

**Comment by nssharmaofficial** [comment]
> have the same issue, i tried debugging and i get an png file back and it gets stuck at a page that says "whoa there, pardner! Your request has been blocked due to an network policy."

@AIlhomov  You need to specify proper user agent in the browser.

**Comment by AIlhomov** [comment]
@nssharmaofficial If i understand it correctly, i specified the [RedditCredential] in config.toml its "user_agent" to my own that i use which is "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:109.0) Gecko/20100101 Firefox/119.0". But it still didnt work, how can it be so?

**Comment by nssharmaofficial** [comment]
> @nssharmaofficial If i understand it correctly, i specified the [RedditCredential] in config.toml its "user_agent" to my own that i use which is "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:109.0) Gecko/20100101 Firefox/119.0". But it still didnt work, how can it be so?

No, that user agent in `[RedditCredential]` is for the reddit bot. You need to specify the user agent of the browser: `context = browser.new_context(user_agent=Mozilla....)`. I hope that helps.

**Comment by AIlhomov** [comment]
@nssharmaofficial Thanks a lot! I managed to solve it by adding that into the code.
